### ISSUE TITLE
shellvar doesn't work properly for export

### ISSUE THREAD

**Issue by devops07** [hercules-team/augeasproviders_shellvar#20]
Want to edit below line in puppet

```javascript
/opt/dtp/etc/dtp.conf
export CHO='NON-PROD'
```
Used shellvar resource, please find the snippet below

```javascript 
shellvar { 'CHO':
  ensure => exported,
  target => '/opt/dtp/etc/dtp_config',
  value  => 'NON-PROD',
}
```

when I run first time it's removing the word "export", got o/p like ```javascriptDTP="NON-PROD"```

If I run second time, getting below o/p
```javascript"Notice: /Stage[main]/Example::Dtp/Shellvar[DTP]/ensure: ensure changed 'present' to 'exported'```
 and the o/p is  ``` export DTP="NON-PROD```

please refer the below url
http://stackoverflow.com/questions/33168034/unable-to-print-augeas-for-own-configuration

**Comment by raphink** [comment]
This is probably a duplicate of https://github.com/hercules-team/augeasproviders_shellvar/pull/10

**Comment by raphink** [comment]
I will merge #10 and you can test if that fixes your issue (but I'm pretty sure it will).

**Comment by raphink** [comment]
Fixed in #10. Will release asap.

**Comment by raphink** [comment]
Released as 2.2.1
